jetty9-base/src/main/jetty-config/jetty.commands

```diff
@@ -1,6 +1,6 @@
 --create-startd
 
---add-to-start=server,webapp,http,deploy,jsp,jstl,http-forwarded,resources,gae
```

Comment:
Some of the config below like disabling the RFC7239 header should probably be part of a GCP module. Is it possible to organize it that way?